libafl_bolts/src/llmp.rs

```diff
@@ -155,7 +152,7 @@ const _LLMP_BIND_ADDR: &str = "0.0.0.0";
 const _LLMP_BIND_ADDR: &str = "127.0.0.1";
 
 /// LLMP Client connects to this address
-const _LLMP_CONNECT_ADDR: &str = "127.0.0.1";
+pub const LLMP_CONNECT_ADDR: &str = "127.0.0.1";
```

Comment:
Why `pub`?

Reply:
i use it from libafl not libafl_bolts

Reply:
Let's rename the guy to `IP_LOCALHOST` and move it out of llmp, then?